content/13.conclusion.md

```diff
@@ -1,6 +1,6 @@
 ## Conclusion {#conclusion}
 
-Deep learning techniques have the potential for wide use in biology and the capacity to meet or exceed the performance of both humans and the current state-of-the art algorithms across a wide range of tasks.
-Beyond simply achieving good predictive performance, deep learning has the potential to inform novel biological insights to fundamentally drive high value research.
+Deep learning techniques have the potential for wide use in biology and the capacity to meet or exceed the performance of both humans and the current state-of-the-art algorithms across a wide range of biological sub-disciplines and tasks.
+Beyond simply achieving good predictive performance, deep learning has the potential to inform novel biological insights to fundamentally drive high-impact research.
 To realize this potential, the use of deep learning as a research tool must be approached as any other tool would be: scientifically and thoughtfully.
-We hope that our tips for applying deep learning to the biological sciences will serve as a starting point for discussion and not as an ending point.
+We hope that our tips for applying deep learning to the biological sciences will serve as a starting point for discussion and considering deep learning for biological research sensibly and responsibly.
```

Comment:
I think this edited sentence may have combined wrong?

How about this? 

We hope that our tips for applying deep learning to the biological sciences will serve as a starting point for discussion and a roadmap for sensible and responsible application.